### DISCUSSION TITLE
Interface to a shopping app like BRING

### DISCUSSION THREAD

**Discussion by krafeiner** [TandoorRecipes/recipes#1333] [Ideas]
t would be nice to have an interface for exporting a recipe to a shopping app. Currently, one of the most popular shopping apps available both on Android and iOS and  is BRING. There is a web UI (https://web.getbring.com/), too.

Some sites like 'chefkoch.de' already offer the option to export a recipe to Bring.

**Comment by tomtjes** [comment]
It's on the roadmap: #1136
